cmd/fleet/serve.go

```diff
@@ -149,7 +149,11 @@ the way that the Fleet server works.
 			var carveStore fleet.CarveStore
 			mailService := mail.NewService()
 
-			ds, err = mysql.New(config.Mysql, clock.C, mysql.Logger(logger))
+			var replicaOpt mysql.DBOption
+			if config.MysqlReadReplica.Address != "" {
```

Comment:
i.e. it considers that a replica has been configured if at least an address has been set (there is no default value set automatically for the replica's address).